hail/python/test/hail/methods/test_skat.py

```diff
@@ -562,7 +560,7 @@ def test_skat_5():
             logistic=(25, 1e-6))._force_count()
 
 
-@test_timeout(local=4 * 60)
```

Comment:
You have to preserve the spark and local timeouts for each of these methods

Reply:
n.b. [default timeout is currently 2 minutes for spark](https://github.com/hail-is/hail/blob/main/build.yaml#L1178)